xchange-bitfinex/src/main/java/info/bitrich/xchangestream/bitfinex/BitfinexStreamingRawService.java

```diff
@@ -0,0 +1,303 @@
+package info.bitrich.xchangestream.bitfinex;
```

Comment:
I've not reviewed this code - I suspect it might move around quite a lot based on my other comments.

Will review properly once the structure is right.